[2024-01-08T08:12:10.134+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetching_cleaning_push_elastic.Fetch_from_Postgresql scheduled__2023-12-28T06:30:00+00:00 [queued]>
[2024-01-08T08:12:10.207+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetching_cleaning_push_elastic.Fetch_from_Postgresql scheduled__2023-12-28T06:30:00+00:00 [queued]>
[2024-01-08T08:12:10.213+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-01-08T08:12:10.215+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2024-01-08T08:12:10.216+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-01-08T08:12:10.283+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): Fetch_from_Postgresql> on 2023-12-28 06:30:00+00:00
[2024-01-08T08:12:10.361+0000] {standard_task_runner.py:52} INFO - Started process 587 to run task
[2024-01-08T08:12:10.408+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetching_cleaning_push_elastic', 'Fetch_from_Postgresql', 'scheduled__2023-12-28T06:30:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_saepul_hilal_DAG.py', '--cfg-path', '/tmp/tmp7sad8djr', '--error-file', '/tmp/tmpo_1ig_ea']
[2024-01-08T08:12:10.414+0000] {standard_task_runner.py:80} INFO - Job 76: Subtask Fetch_from_Postgresql
[2024-01-08T08:12:10.819+0000] {task_command.py:371} INFO - Running <TaskInstance: fetching_cleaning_push_elastic.Fetch_from_Postgresql scheduled__2023-12-28T06:30:00+00:00 [running]> on host 0df44d59871b
[2024-01-08T08:12:11.373+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=saepul
AIRFLOW_CTX_DAG_ID=fetching_cleaning_push_elastic
AIRFLOW_CTX_TASK_ID=Fetch_from_Postgresql
AIRFLOW_CTX_EXECUTION_DATE=2023-12-28T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-28T06:30:00+00:00
[2024-01-08T08:12:15.472+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-01-08T08:12:16.308+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=fetching_cleaning_push_elastic, task_id=Fetch_from_Postgresql, execution_date=20231228T063000, start_date=20240108T081210, end_date=20240108T081216
[2024-01-08T08:12:16.558+0000] {local_task_job.py:227} WARNING - State of this instance has been externally set to success. Terminating instance.
